sdk/communication/Azure.Communication.Administration/src/PhoneNumberAdministrationClient.cs

```diff
@@ -60,25 +61,16 @@ protected PhoneNumberAdministrationClient()
         /// <returns> A <see cref="AsyncPageable{AcquiredPhoneNumber}"/>. </returns>
         public virtual AsyncPageable<AcquiredPhoneNumber> GetAllPhoneNumbersAsync(string? locale = null, CancellationToken cancellationToken = default)
         {
-            using DiagnosticScope scope = ClientDiagnostics.CreateScope($"{nameof(PhoneNumberAdministrationClient)}.{nameof(GetAllPhoneNumbers)}");
-            scope.Start();
-
-            try
-            {
-                return PageResponseEnumerator.CreateAsyncEnumerable(async nextLink =>
+            return ClientDiagnostics.RunScoped(() =>
```

Comment:
Use expression body instead of return, same for all the other places.

Reply:
You can format it in a way that the lambda body aligns nicely with a normal function body.

Reply:
Dominik
I did use expression body but did not like the way it ended up
![image](https://user-images.githubusercontent.com/30868871/101203395-703f1300-361f-11eb-8f6b-ff3f587fb8ff.png)

In my opinion it does not make code easier to read but rather more complex. I like to use expression body for a single liners.